smithy.yaml

```diff
@@ -1,11 +1,12 @@
 project: dart
 language: dart
 
-# dart 1.22.0, built from https://github.com/Workiva/smithy-runner-dart/tree/0.0.9
-runner_image: drydock-prod.workiva.org/workiva/smithy-runner-dart:124921
+# This image is created using https://github.com/Workiva/smithy-runner-generator
+runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:203768 # Dart 1.24.2
 
 script:
-  - pub get
+  - pub get --packages-dir
+  - xvfb-run -s '-screen 0 1024x768x24' pub run dart_dev test --web-compiler=dartdevc -p chrome -p vm
```

Comment:
It's not required, I just added it to ensure that all tests were run (vm and browser). Since testing in the VM is fast, I don't think it's a big deal to leave it in.